FizzBuzz/part2.py

```diff
@@ -0,0 +1,34 @@
+

```

Comment:
Leaving a comment for visibility here. Could you move the `.gitignore` file outside the `.idea` folder and add the `.idea` folder to it so it doesn't get committed?